src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -29,31 +29,57 @@ Copyright (c) .NET Foundation. All rights reserved.
     included in the generated deps.json.
     ============================================================
     -->
+  <UsingTask TaskName="FindNativeDeps" AssemblyFile="$(ILLinkTasksAssembly)" />
   <Target Name="_ILLink"
```

Comment:
Nit: I think you should rename this to ILLink without the underscore. I think the convention used seems to be that all targets in the `DependsOnTargets` for the `ComputeAndCopyFilesToPublishDirectory` don't have underscores. Subtargets like `_RunILLink` can have the underscore